src/main/java/bio/terra/service/auth/iam/IamProviderInterface.java

```diff
@@ -279,4 +280,18 @@ void overwriteGroupPolicyEmails(
   String signUrlForBlob(
       AuthenticatedUserRequest userReq, String project, String path, Duration duration)
       throws InterruptedException;
+
+  /**
+   * Create a managed resource group in Azure that is associated with a billing profile.
```

Comment:
This endpoint does not actually create the MRG, so I'd update the documentation to something like the following to avoid confusion (I used the [Sam endpoint's](https://sam.dsde-prod.broadinstitute.org/#/Azure/createManagedResourceGroup) documentation as a reference).

```suggestion
   * Associate billing profile with an Azure managed resource group.
```